core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -1011,10 +1021,21 @@ private void processReply(RaftClientReply reply, String msgToUser) throws IOExce
               ? reply.getException()
               : new IOException(String.format("reply <%s> failed", reply));
       LOG.error("{}. Error: {}", msgToUser, ioe);
-      throw new IOException(msgToUser);
+      IOException exception = new IOException(msgToUser);
+      mTransferMsgs.add(TransferLeaderMessage.newBuilder()
```

Comment:
I didn't find where is responsible for remove or clear element of `mTransferMsgs `, would you like to have a double check?